steps/incidents-create/spec.schema.json

```diff
@@ -54,11 +54,15 @@
     },
     "state": {
       "type": "integer",
-      "description": "State of the incident"
+      "description": "State of the incident (e.g. New)"
     }
   },
   "required": [
-    "shortDescription"
+    "shortDescription",
+    "description",
+    "priority",
+    "impact",
+    "state"
```

Comment:
I think only the `shortDescription` (and perhaps `description`) are required by servicenow. Do you want to make them all required like this, or catch 422 responses like https://github.com/relay-integrations/relay-puppet/blob/1e0c2b67e778a26fcd91ed6bf2680f120e9a67d8/steps/task-run-start/step.py#L15-L21 and make them not required?

Reply:
I say let's move forward with them as required for now.